### PR TITLE
Milestone 2 - Add Display and Button components

### PR DESCRIPTION
**Pull request by Stephanie041996**
-defining the basic React components 
-Added a root component called App which will contain a Display component responsible for displaying the outcome of the calculations, and a ButtonPanel component which will just be a container for all the buttons.
-Each button in ButtonPanel will implement the Button component which represents a single button.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="My-App">
+      <h1>Calculator App</h1>
+      <Display />
+      <ButtonPanel />
+    </div>
```
Comment (taiwo2) [inline src/components/App.js]: should be wrapped with React.Fragment short syntax

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <div>
+          <button type="button">1</button>
+          <button type="button">2</button>
+          <button type="button">3</button>
+          <button type="button">+</button>
+        </div>
+        <div>
+          <button type="button">0</button>
+          <button type="button">.</button>
+          <button type="button">=</button>
```
Comment (taiwo2) [inline src/components/ButtonPanel.js]: Each of the button should be  a `Button Component` not direct HTML `button` tag

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@Stephanie041996 congratulations on this project, most of the project milestone requirement is met, your code is unique indeed. however, changes are required, and kindly find my inline comments 
###Highlights 
Git flow 👍🏻 
project structure 👍🏻 
### To Improve 
`App` component

- [x] has children components wrapped in `React.Fragment`  shot syntax.
- [ ] it renders the calculator panel using `Button component` not the HTML button tag directly 

### once a specific change is made kindly request for review

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (Stephanie041996) [reply src/components/ButtonPanel.js]: Thanks, Taiwo for the review, I have implemented the changes

**Review by whiz25 — APPROVED** [review]
## approved 👍🏿 👍🏿👍🏿
Nice work making the requested changes.

### highlights 🌵 🌵 🌵 
- use of React.Fragment shorthand
- use of Button component instead of the HTML button tag


### Martin Afani
#### Slack: whiz25

**Pull request merged**